hypervisor/init_comm.go

```diff
@@ -79,6 +79,11 @@ func ReadVmMessage(conn *net.UnixConn) (*DecodedMessage, error) {
 		res = append(res, buf[:nr]...)
 		read = read + nr
 
+		// if condition satisified, message length may wrong , read again
+		if nr < want && read < needRead {
+			return ReadVmMessage(conn)
+		}
+
```

Comment:
Actually, you don't need to create a new pr for qemu lazy, just use this one.
just create a new pr for ReadVmMessage().